package.json

```diff
@@ -23,7 +23,9 @@
     "Aappo Ålander",
     "Joonas Kaski",
     "Sami Korpela",
-    "Valtteri Laukkanen"
+    "Valtteri Laukkanen",
+    "Ilkka Pättikangas",
+    "Mikko Riippi"
```

Comment:
I think this is good thing to do here now; no need for separate PR in my opinion.